### ISSUE TITLE
non-recoverable error with on_gui_value_changed v2.1.0

### ISSUE THREAD

**Issue by Dairyman** [DarkShadow44/resourcehighlighter-dark#14]
```
The mod Resource Highlighter (Dark's Version) (2.1.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event resourcehighlighter-dark::on_gui_value_changed (ID 89)
__resourcehighlighter-dark__/control.lua:471: attempt to index field 'min_resource_label' (a nil value)
stack traceback:
	__resourcehighlighter-dark__/control.lua:471: in function <__resourcehighlighter-dark__/control.lua:467>
```

You need to check if it is your GUI element. Error here happened as I changed the `Multiplier` value in [Rate Calculator](https://mods.factorio.com/mod/RateCalculator).

**Comment by loneguardian** [comment]
can confirm this, was changing the slider in Factory Planner and it crashed with a similar message:

```
The mod Resource Highlighter (Dark's Version) (2.1.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event resourcehighlighter-dark::on_gui_value_changed (ID 89)
__resourcehighlighter-dark__/control.lua:471: attempt to index field 'min_resource_label' (a nil value)
stack traceback:
	__resourcehighlighter-dark__/control.lua:471: in function <__resourcehighlighter-dark__/control.lua:467>
```

**Comment by ajklotz** [comment]
I had the same crash
```
The mod Resource Highlighter (Dark's Version) (2.1.0) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event resourcehighlighter-dark::on_gui_value_changed (ID 89)
__resourcehighlighter-dark__/control.lua:471: attempt to index field 'min_resource_label' (a nil value)
stack traceback:
	__resourcehighlighter-dark__/control.lua:471: in function <__resourcehighlighter-dark__/control.lua:467>
```

**Comment by DarkShadow44** [comment]
Sorry for that, will be fixed with 91fc2a6497e09257f2e3bb167afd796620096aab!
